astropy/coordinates/solar_system.py

```diff
@@ -43,28 +43,42 @@
 SOLAR_SYSTEM_BODIES = tuple(BODY_NAME_TO_KERNEL_SPEC.keys())
 
 
-def _download_spk_file(url=('http://naif.jpl.nasa.gov/pub/naif/'
-                            'generic_kernels/spk/planets/de430.bsp'),
-                       show_progress=True):
+def set_kernel_url(url='http://naif.jpl.nasa.gov/pub/naif/generic_kernels'
```

Comment:
I thought that at first, but then decided definitely not.  It's "science state", in that the actual results you get depend on which kernel you are using.  Configuration items are intended for things that do *not* affect science state, on the theory that it's bad to implicitly set anything that actually affects the scientific results.

Put another way, in an ipython notebook we want it to always be necessary to put this at the top to specify the kernel, rather than counting on some configuration file hidden in the server that the notebook is running on.